src/pages/extending-chat-widget/customer-sdk/index.mdx

```diff
@@ -28,7 +28,7 @@ If you want to dive deeper into the LiveChat Platform, you might find the [Platf
 ## About LiveChat Customer SDK
 
 We provide an asynchronous API. Most methods return a [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). You can subscribe to the emitted events with the `on` and `off` methods.
-Not familiar with promises? Read <a href="https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/Promise">this article</a> to learn more.
+Not familiar with promises? Read <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise">this article</a> to learn more.
```

Comment:
This case is connected with [this issue](https://livechat.slack.com/archives/C2XMTMQ5S/p1663848633870359) raised on #platform-docs. I agree with Damian that the link should be changed for the one he proposed:) 
Change has been made in v3.0;2.0;1.0